pkg/apiserver/rest/dualwriter_mode1.go

```diff
@@ -28,35 +32,123 @@ func NewDualWriterMode1(legacy LegacyStorage, storage Storage) *DualWriterMode1
 // Create overrides the behavior of the generic DualWriter and writes only to LegacyStorage.
 func (d *DualWriterMode1) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
 	ctx = klog.NewContext(ctx, d.Log)
-	return d.Legacy.Create(ctx, obj, createValidation, options)
+	var method = "create"
+
+	startStorage := time.Now().UTC()
+	_, err := d.Storage.Create(ctx, obj, createValidation, options)
```

Comment:
Shall we run each in two separate goroutines (with a sync.WaitGroup to ensure we return when both complete )?

Reply:
interesting, so we could call them in parallel? That could potentially help with latency. Good idea :)